tests/prague/eip7623_increase_calldata_cost/test_transaction_validity.py

```diff
@@ -70,6 +70,7 @@
 @pytest.mark.parametrize(
     "to",
     [
+        pytest.param(Address(1), id="to_address"),
```

Comment:
This is actually a precompile, so maybe we have to modify the `to` fixture in `tests/prague/eip7623_increase_calldata_cost/conftest.py` to accept maybe a string literal or some other sentinel value to transform it into `pre.fund_eoa(amount=0)`.

Reply:
lol, thanks! Fixed.